src/main/java/org/folio/rest/impl/RoutingListsAPI.java

```diff
@@ -31,9 +31,6 @@ public void getOrdersStorageRoutingLists(String query, String totalRecords, int
   @Validate
   public void postOrdersStorageRoutingLists(RoutingList entity, Map<String, String> okapiHeaders,
                                             Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-    if (StringUtils.isBlank(entity.getId())) {
-      entity.setId(UUID.randomUUID().toString());
-    }
```

Comment:
after removing this part, since we deleted required field, where uuid is being generated.

Reply:
I mean in case of FE don't send generated uuid

Reply:
PgClient does it